test/performance/tests/test_util/config.py

```diff
@@ -39,9 +39,10 @@
     os.getenv("TEST_KUBE_BURNER_URL")
     or "https://github.com/kube-burner/kube-burner/releases/download/v1.2/kube-burner-1.2-Linux-x86_64.tar.gz"
 )
-
 # Global kube-burner invocation timeout.
 KUBE_BURNER_TIMEOUT = os.getenv("TEST_KUBE_BURNER_TIMEOUT") or "10m"
+# The number of kube-burner invocations.
+KUBE_BURNER_ITERATIONS = int(os.getenv("TEST_KUBE_BURNER_ITERATIONS") or 1)
```

Comment:
```suggestion
KUBE_BURNER_ITERATIONS = int(os.getenv("TEST_KUBE_BURNER_ITERATIONS") or 15)
```
Let's set the default higher please, repetition adds to the quality of our results :)

Reply:
The github job sets ``TEST_KUBE_BURNER_ITERATIONS`` to 15, I used a default of 1 here just so that if someone runs ``tox -e performance`` locally they won't have to wait for 15 iterations.

I can change this to 15 though.